doc/anaconda.md

```diff
@@ -0,0 +1,73 @@
+Cockpit Storage in Anaconda Mode
+================================
+
+Anaconda (the OS Installer) can open the Cockpit "storaged" for
+advanced setup of the target storage devices. When this is done,
+storaged is in a special "Anaconda mode" and behaves significantly
+different.
+
+In essence, the storaged page restricts itself to working with the
```

Comment:
While most real world Live installations will indeed happen from USB stick, I just wanted to point out there are technically other ways to boot a Live image, but given that there will be support for hiding arbitrary devices, that should not be a problem hopefully.